designcompose/src/main/java/com/android/designcompose/squoosh/SquooshRender.kt

```diff
@@ -333,12 +332,13 @@ private fun squooshTextRender(
     drawContext.canvas.save()
     drawContext.canvas.translate(0.0f, verticalCenterOffset)
 
-    val strokeWidth = style.nodeStyle.stroke.get().stroke_weight.toUniform() * density.density
+    val strokeWidth =
+        (style.nodeStyle.strokeOrNull?.strokeWeightOrNull?.toUniform() ?: 0f) * density.density
     // Only drop shadow is supported now.
-    val shadow = style.nodeStyle.text_shadow.getOrNull()
-    val blurRadius = (shadow?.blur_radius ?: 0f) * density.density
-    val xOffset = (shadow?.offset_x ?: 0f) * density.density
-    val yOffset = (shadow?.offset_y ?: 0f) * density.density
+    val shadow = style.nodeStyle.textShadowOrNull
+    val blurRadius = (shadow?.blurRadius ?: 0f) * density.density
+    val xOffset = (shadow?.offsetX ?: 0f) * density.density
+    val yOffset = (shadow?.offsetY ?: 0f) * density.density
```

Comment:
Should be able to handle this with proto returning defaults:

```suggestion
    val shadow = style.nodeStyle.textShadow
    val blurRadius = shadow.blurRadius * density.density
    val xOffset = shadow.offsetX * density.density
    val yOffset = shadow.offsetY * density.density
```

Reply:
Done